### PR TITLE
cargo fmt

### PR DESCRIPTION
**Pull request by jean-santos**
Just a cargo fmt

### CODE REVIEW

File: src/parser.rs [hunk 1]
```diff
@@ -73,8 +93,18 @@ pub fn parse_and_generate_code(schema_file_contents: String, config: &Generation
         }
     }
 
+    let mut tables_hm = std::collections::HashMap::new();
+
     for table in tables.iter_mut() {
-        table.generated_code = code::generate_table(table.clone(), config);
+        // table.generated_code = code::generate_table(table.clone(), config);
+        tables_hm.insert(table.name.clone(), table);
```
Comment (AnthonyMichaelTDM) [inline src/parser.rs:100]: this looks like a bit more than just a fmt change, will this modify the behavior of the program?

File: src/parser.rs [hunk 2]
```diff
@@ -73,8 +93,18 @@ pub fn parse_and_generate_code(schema_file_contents: String, config: &Generation
...
+    }
+
+    for table in tables_hm.iter_mut() {
+        // **table.1 = code::generate_table(table.clone(), config, &tables_hm);
+
+        // (table.1).generated_code = String::new();
+        table.1.generated_code = String::new();
```
Comment (AnthonyMichaelTDM) [inline src/parser.rs:107]: this looks like a bit more than just a fmt change, will this modify the behavior of the program?

**Review by AnthonyMichaelTDM — CHANGES_REQUESTED** [review]
Other than my previous comment, this looks good to me

File: src/parser.rs (hunk 2 above)
Comment (jean-santos) [reply src/parser.rs:107]: That's awkward! This shouldn't be here, my mistake!
Thanks for the review by the way! Gonna send the fixup for this!

**Review by AnthonyMichaelTDM — APPROVED** [review]

File: src/parser.rs (hunk 2 above)
Comment (AnthonyMichaelTDM) [reply src/parser.rs]: Yeah, no worries.

Looks better on my activity graph if I'm contributing in more ways than just commits, lol.

**Comment by Wulf** [comment]
@jean-santos thanks for this and @AnthonyMichaelTDM thanks for reviewing it

**Pull request merged**